app-editors/vis/Manifest

```diff
@@ -1 +1,2 @@
 DIST vis-0.3.tar.gz 334216 SHA256 bed8ed32ef4f81bc4fd28e297ea8edc80027ac6923b9d3b5bd955d24899c7077 SHA512 e1ca009bd7569f22dacf7464791f9f4499695d15d6e38e1be491d39e7ebe0530de643be5c1e7647e9beb7969a17efab6545d3e719fe8cf311562e89d636bba62 WHIRLPOOL a0ddd69b88b4131d4b9c2ba5494e255f20cd99f91998e4ad4fcec3e0298e39d9319571201b8ae4c8b4d21da78b99d865d72bfb534024a89863d66b9968e562c8
+DIST vis-test-0.1.tar.gz 95274 SHA256 78c2c4416ef284e2362bb0e761ee9c4add106d122ac44b341c6bcf449f0b17a6 SHA512 433a67892f27eb01f3ea6224e1ded393cd949b4d60f7c14656fdd526c02d2b309f63d39887d8d7ba11cbad3e183ab3cd1a913ac40aa29eadb7fa700eb76424b6 WHIRLPOOL 45d257cdc625d249e45d8581f11c976c4957812a41274943af622521e6a599bc012809fa17651dcc278579b1f0ef2d40468c746080cf2f82b52854854073e1e8
```

Comment:
Squash this into the original commit needing this change. Don't introduce interim broken commits.